### PR TITLE
Add kicking stats to NCAAF players

### LINKED ISSUE #373: Add Kicking table stats for NCAAF players
Currently the `"kicking"` table is not being read for (slightly) more advanced kicking stats for NCAAF players. I have PR coming to fix this so no need to start working.

### PR DESCRIPTION
**Pull request by AidanGlickman**
Fixes #373 

Adds kicking stats (attempts and percentage) to NCAAF players.

### CODE REVIEW

File: sportsreference/ncaaf/player.py [hunk 1]
```diff
@@ -501,10 +508,41 @@ def extra_points_made(self):
...
+        """
+        Returns an ``int`` of the total number of extra points the player
+        attempted.
+        """
+        return self._extra_points_attempted
+
+    @_float_property_decorator
+    def extra_point_percentage(self):
+        """
+        Returns an ``float`` of the percentage of extra points the player made.
```
Comment (roclark) [inline sportsreference/ncaaf/player.py:522]: Returns **a** `float` (as opposed to `an`).

File: sportsreference/ncaaf/player.py [hunk 2]
```diff
@@ -501,10 +508,41 @@ def extra_points_made(self):
...
+        """
+        Returns an ``int`` of the total number of field goals the player
+        attempted from any distance.
+        """
+        return self._field_goals_attempted
+
+    @_float_property_decorator
+    def field_goal_percentage(self):
+        """
+        Returns an ``float`` of the percentage of field goals the player made.
```
Comment (roclark) [inline sportsreference/ncaaf/player.py:545]: Returns **a** `float` again here.

File: sportsreference/ncaaf/player.py [hunk 3]
```diff
@@ -501,10 +508,41 @@ def extra_points_made(self):
...
+        from any distance.
```
Comment (roclark) [inline sportsreference/ncaaf/player.py:546]: Same as before, adding a note about the percentage range would be useful.

File: sportsreference/ncaaf/player.py [hunk 4]
```diff
@@ -68,6 +68,7 @@ class AbstractPlayer:
         page. If the player appears in multiple tables, all of their
         information will appear in one single string concatenated together.
     """
+
```
[the remark this answers is not in the document]
Comment (AidanGlickman) [reply sportsreference/ncaaf/player.py:71]: I think this came from `autopep8`, which I'm pretty sure is based on pycodestyle. Removed.

**Comment by AidanGlickman** [comment]
Hi @roclark thanks for the kind words! I squashed the two commits and fixed the specific issues you mentioned. I think the testing suite needs to be changed a bit more structurally to account for these specific stats as the example html file in the integration test does not seem to have the `kicking` table. That probably warrants a new PR.

**Comment by AidanGlickman** [comment]
git is acting up now... gonna need to figure out why it unsquashed my commits

**Comment by AidanGlickman** [comment]
not sure what happened here, but the commits seem to be empty here now? opened again as #381 because I couldn't figure it out. Please feel free to fix however you see fit.

**Pull request closed without merging**